app/src/main/java/com/simprints/intentlauncher/ui/intent/composables/CoreFieldsForm.kt

```diff
@@ -51,6 +52,13 @@ fun CoreFieldsForm(
                     .weight(1f),
             )
         }
+        PropertyField(
```

Comment:
Also, it would be nice to have some indication and validation that metadata must be a valid JSON. Currently, it just crashes.

Reply:
About hiding the field - implemented it but upon trying it out - it will be much more irritating for the people that do use it. As the screen has enough space for all fields I suggest we leave it present at all times and only change if it proves to be a problem.
About JSON error handling - added it.

Reply:
Not on my device :D Would it be hard to persistthe state of the expanded fields somehow?

Reply:
I was thinking about this but didn't want to get too distracted from the core GG2 stuff. How about this - I'll push the show/hide changes and outsource the state persistence to you?

Reply:
Maybe on the next lego days :D